AssemblyAutoBoard/AssemblyAutoBoard.ino

```diff
@@ -263,228 +256,260 @@ void loop() {
-         ManualFeed - Ignore other variables and trigger on button press
+    // active Mode start.  Machine will read sensors and run relays.
+    if ((active == 1) && (runCheck == 1)) {
+      digitalWrite(solenoidArray[7], HIGH);
+      manualFeed = digitalRead(manualButton);
+      feedLoop = digitalRead(sensorArray[2]);
+      feedCheck = digitalRead(sensorArray[1]);
+      /* feedLoop - Check your main cycle sensor
+         error - Check to see if the machine went into error Mode
+         secStart - Restart machine after feedCheck activates again
+         feedCheck - Check Feed station for material.
+         manualFeed - Ignore other variables and trigger on button press
       */
-      if (((FeedLoop == LOW) && (Error == 0)) || ((SecStart == 1) && (FeedCheck == LOW)) || (ManualFeed == HIGH)) {
-        if (FeedNext == 0) {
+      if (((feedLoop == LOW) && (partError == 0)) || ((secStart == 1) && (feedCheck == LOW)) || (manualFeed == HIGH)) {
+        if ((feedNext == 0) && (currentTime - previousTimer1 >= sysArray[6])) {
           // FEED ACTIVATED
           Serial.println("Feed Cycle Activated");
           lcd.setCursor(0, 2);
           lcd.print("Feed Reset:");
           //Start counting time for TimeKeepr function
-          if (LogicCount == 0) {
+          if (logicCount == 0) {
             precountTime = currentTime;
           }
           //Check Feed station for material.
-          FeedCheck = digitalRead(SensorArray[1]);
-          if ((FeedCheck == HIGH) && (SecStart != 1)) {
+          feedCheck = digitalRead(sensorArray[1]);
+          if ((feedCheck == HIGH) && (secStart != 1)) {
             Serial.println("ERROR: Hanger Rack NOT full.");
             lcd.setCursor(0, 3);
             lcd.print("ERROR: Hanger Rack");
             preLCDClear = currentTime;
-            Error = 1;
-            SecStart = 1;
+            partError = 1;
+            secStart = 1;
             lcd.setCursor(11, 2);
             lcd.print("ON ");
           }
           else {
             //Add one to logic count
-            LogicCount++;
-            SecStart = 0;
+            logicCount++;
+            secStart = 0;
             lcd.setCursor(11, 2);
             lcd.print("OFF");
-            Error = 0;
+            partError = 0;
             lcd.setCursor(0, 1);
             lcd.print("SC: ");
             lcd.setCursor(4, 1);
-            lcd.print(LogicCount);
+            lcd.print(logicCount);
             lcd.print("  ");
-            digitalWrite(PanelLed1, HIGH);
-            digitalWrite(ErrorLed, LOW);
-            FeedNext = 1;
+            digitalWrite(panelLed1, HIGH);
+            digitalWrite(errorLed, LOW);
+            feedNext = 1;
+            previousTimer1 = currentTime;
+          }
+        }
+        if (mpsEnable >= 1) {
+          if ((currentTime - previousTimer1 <= sysArray[6]) && (currentTime - previousTimer1 >= buttonWait) && (manualFeed == LOW)) {
```

Comment:
Should consider using a separate changeable variable for the sensor ignore period.  
(sysArray[8])